Ginger/GingerTest/APIModelLib/POMsTest.cs

```diff
@@ -101,7 +101,7 @@ public void ValidatePOMScreenshotWasTaken()
         public void ValidateLearnedItems()
         {
             //Act
-            ElementInfo EI1 = mLearnedPOM.MappedUIElements.Where(x => x.ElementName == "Mexico INPUT.RADIO" && x.ElementTypeEnum == eElementType.RadioButton).FirstOrDefault();
+            ElementInfo EI1 = mLearnedPOM.MappedUIElements.Where(x => x.ElementName == "Mexico INPUTRADIO" && x.ElementTypeEnum == eElementType.RadioButton).FirstOrDefault();
             ElementInfo EI2 = mLearnedPOM.MappedUIElements.Where(x => x.ElementName == "id123 input" && x.ElementTypeEnum == eElementType.TextBox).FirstOrDefault();
 
             //Assert  
```

Comment:
i'm still missing new Test method with Asserts which checks new added ElementInfo.OptionalValuesObjectsList and ElementInfo.OptionalValuesObjectsListAsString content post learning

Reply:
Added